etc/eslint/rules/typescript.js

```diff
@@ -0,0 +1,2526 @@
+* @see [import/order]{@link https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md}
+*
+* @example
+* // Given the configuration above, the following order is enforced:
+* // 1. Built-in modules (node)
+* // 2. External modules
+* // 3. Internal modules
+* // 4. Modules from a parent directory
+* // 5. Sibling modules from the same or a sibling's directory
+* // 6. Index of the current directory
+*
+* // Bad...
+* import sibling from './sibling';
+* import fs from 'fs';
+*
+* // Good...
+* import fs from 'fs';
+* import sibling from './sibling';
+*/
+rules[ 'import/order' ] = [
+	'error',
+	{
+		'alphabetize': {
+			'caseInsensitive': false,
+			'order': 'ignore'
+		},
+		'newlines-between': 'ignore',
+		'groups': [
+			[
+				'builtin',
+				'external',
+				'internal',
+				'unknown',
+				'object',
+				'type'
+			],
+			'parent',
+			[
+				'sibling',
+				'index'
+			]
+		],
+		'distinctGroup': false,
+		'pathGroupsExcludedImportTypes': [],
+		'pathGroups': [
+			{
+				'pattern': './',
+				'patternOptions': {
+					'nocomment': true,
+					'dot': true
+				},
+				'group': 'sibling',
+				'position': 'before'
+			},
+			{
+				'pattern': '.',
+				'patternOptions': {
+					'nocomment': true,
+					'dot': true
+				},
+				'group': 'sibling',
+				'position': 'before'
+			},
+			{
+				'pattern': '..',
+				'patternOptions': {
+					'nocomment': true,
+					'dot': true
+				},
+				'group': 'parent',
+				'position': 'before'
+			},
+			{
+				'pattern': '../',
+				'patternOptions': {
+					'nocomment': true,
+					'dot': true
+				},
+				'group': 'parent',
+				'position': 'before'
+			}
+		]
+	}
+];
+
+/**
+* Disables the rule for indentation.
+*
+* @name indent
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [indent]{@link https://eslint.org/docs/rules/indent}
+*/
+rules[ 'indent' ] = 'off';
+
+/**
+* Disables the JSDoc rule for checking alignment.
+*
+* @name jsdoc/check-alignment
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [check-alignment]{@link https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-check-alignment}
+*/
+rules[ 'jsdoc/check-alignment' ] = 'off';
+
+/**
+* Disables the JSDoc rule for checking indentation.
+*
+* @name jsdoc/check-indentation
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [check-indentation]{@link https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-check-indentation}
+*/
+rules[ 'jsdoc/check-indentation' ] = 'off';
+
+/**
+* Disables the JSDoc rule for requiring a newline after the description.
+*
+* @name jsdoc/newline-after-description
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [newline-after-description]{@link https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-newline-after-description}
+*/
+rules[ 'jsdoc/newline-after-description' ] = 'off';
```

Comment:
Does this rule still exist?

Reply:
No; removed. I used the [tslint-to-eslint-config](https://github.com/typescript-eslint/tslint-to-eslint-config) tool to aid in the migration, so a bit surprised that I ended up with a removed lint rule in here.